backend/.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   parserOptions: {
     project: 'tsconfig.json',
     sourceType: 'module',
+    tsconfigRootDir: __dirname,
```

Comment:
why wee need this line ?
could you explain this command

Reply:
because eslint can't find tsconfig in my computer so this command will tell eslint where is tsconfig exactly are